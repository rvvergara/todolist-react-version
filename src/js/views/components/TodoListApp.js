import React from 'react';
import ProjectList from './ProjectList';
import Todos from './Todos';

const TodoListApp = () => (
  <div id="todo-app">
    <header>
      <h1 className="header__h1">Todo List App</h1>
    </header>
    <ProjectList />
    <Todos />
  </div>
);

export default TodoListApp;
