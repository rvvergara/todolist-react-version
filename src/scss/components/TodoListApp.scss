#todo-app {
  display: flex;
  flex-direction: column;
}

header {
  background: $strong-purple;
  color: #fff;
}