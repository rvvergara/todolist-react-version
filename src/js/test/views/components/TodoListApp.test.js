import React from 'react';
import { shallow } from 'enzyme';
import TodoListApp from '../../../views/components/TodoListApp.js';

test('TodoListApp should render correctly', () => {
  expect(shallow(<TodoListApp />)).toMatchSnapshot();
});
