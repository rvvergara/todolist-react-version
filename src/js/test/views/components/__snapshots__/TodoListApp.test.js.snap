// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoListApp should render correctly 1`] = `
<div
  id="todo-app"
>
  <ConnectFunction />
  <ConnectFunction />
</div>
`;
